images/build/Dockerfiles/vso.focal.Dockerfile

```diff
@@ -256,6 +249,9 @@ RUN buildDir="/opt/tmp/build" \
     && mkdir -p /home/codespace/.maven/current \
     && ln -sfn /opt/maven/$MAVEN_VERSION /home/codespace/.maven/current \
     && npm install -g lerna \
+    && PATH="$PATH:/opt/php/lts/bin" \
+    && wget http://pear.php.net/go-pear.phar \
+    && php go-pear.phar \
```

Comment:
What is these two lines for?

Reply:
It is installing `pecl` which is been used in next few lines. 
It looked like php 8 did not install with the `--with-pear` flag, hence, a workaround.